sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2.scala

```diff
@@ -56,8 +56,9 @@ object HiveThriftServer2 extends Logging {
    *                    the call logs the error and exits the JVM with exit code -1. When false, the
    *                    call throws an exception instead.
    */
+  @Since("3.5.2")
```

Comment:
4.0.0?

Reply:
Oh, I thought https://github.com/apache/spark/pull/44575 was backported.

Reply:
To @dragqueen95 , could you make a follow-up to fix this to `4.0.0` since SPARK-46575 is not in branch-3.5?

Reply:
Raised https://github.com/apache/spark/pull/45737